frontend/test/parsing/testParseModuleInclude.cpp

```diff
@@ -17,24 +17,14 @@
  * limitations under the License.
  */
 
+#include "test-parsing.h"
+
 #include "chpl/parsing/parsing-queries.h"
-#include "chpl/resolution/resolution-queries.h"
-#include "chpl/resolution/scope-queries.h"
+//#include "chpl/resolution/resolution-queries.h"
+//#include "chpl/resolution/scope-queries.h"
```

Comment:
Deleting is probably fine here if they're not in use?

Reply:
Yes, of course. I will do.